### ISSUE TITLE
__init__() got an unexpected keyword argument 'pretrained'

### ISSUE THREAD

**Issue by sssruhan1** [bearpaw/pytorch-pose#1]
Hi,

1. I was trying to test out a trained model on validation set . When I use argument
-e -a hg1  -c checkpoint/mpii/hg1 -d --pretrained
it's giving me error : __init__() got an unexpected keyword argument 'pretrained'

It looks like it's from HourglassNet() __init__() function, where there's no pretrained argument. 

Did I do something wrong? Or should I just use the --test-batch argument in the command line?

I'm a newbie for pytorch :(  

2. Is there a pretrained stacked hourglass model I can test out in PyTorch? Since the original paper provided a torch version and it seems a bit pain to try to load it into PyTorch.

Thanks in advance.

**Comment by bearpaw** [comment]
@sssruhan1 Hi, it's my fault. The `--pretrained` arg is never used in this code. 

I've updated the code and the README. This update also includes matlab scripts to compute the PCKh@0.5 score on the MPII validation set. Additionally, I release a 4-stack hourglass model for your reference. 

Please check the latest update, and read the [Readme](https://github.com/bearpaw/pytorch-pose#testing) for how to do testing and evaluation.

**Comment by sssruhan1** [comment]
Thank you very much! I've also implemented a python version of compute PCKh@0.5. I will test it out and do a pull request. 
Nice work BTW!
